package.json

```diff
@@ -3,9 +3,13 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "immutable": "^3.8.2",
-    "react": "^16.11.0",
+    "bootstrap": "^4.3.1",
+    "immutable": "3.8.2",
+    "prop-types": "^15.7.2",
+    "react": "^16.10.2",
+    "react-bootstrap": "^1.0.0-beta.14",
     "react-dom": "^16.11.0",
+    "react-dropzone": "^10.1.10",
```

Comment:
should we use-exact to keep consistent with web-ui practices?

Reply:
I agree I will change this.